# Definition of ROS service LocGetErrorLevel for sick localization.
#
# ROS service LocGetErrorLevel queries the error status
# by sending cola command ("sMN GetErrorLevel").
#
# See Telegram-Listing-v1.1.0.241R.pdf for further details about 
# Cola telegrams and this command.

#
# Request (input)
#

---

#
# Response (output)
#

uint8 level # The error level of the software: 0 = no error, 1 = warning, 2 = error
string description # A text describing the current error. Max length = 1024 
bool success # true: system ready response received from localization controller, false: system not ready or service failed (timeout or error status from controller)

